package.json

```diff
@@ -34,6 +34,7 @@
   "homepage": "https://github.com/bencoveney/barrelsby#readme",
   "dependencies": {
     "@types/yargs": "^17.0.10",
+    "lilconfig": "^3.1.3",
```

Comment:
I used lilconfig as a more lightweight alternative to cosmiconfig. This opens up the door for barrelsby to support many different approaches to configuration loading/overriding.